lib/filter-schema.js

```diff
@@ -109,12 +109,31 @@ async function filterSchema (graphQLSchema, policies, context) {
             operation: { kind: 'OperationDefinition', operation: 'query' },
             variableValues: {}
           }
-          // The null parameters are: https://graphql.org/learn/execution/#root-fields-resolvers
-          // - parent: it is not possible know it since the resolver is not executed yet
-          // - args: it is not expected that the introspection query will have arguments for the directives policies
-          canShowDirectiveField = await policyFunction(fieldPolicy, null, null, context, info)
-          if (canShowDirectiveField instanceof Error || !canShowDirectiveField) {
-            canShowDirectiveField = false
+
+          if (Array.isArray(fieldPolicy)) {
+            const promises = []
+
+            for (const item of fieldPolicy) {
+              // The null parameters are: https://graphql.org/learn/execution/#root-fields-resolvers
+              // - parent: it is not possible know it since the resolver is not executed yet
+              // - args: it is not expected that the introspection query will have arguments for the directives policies
+              promises.push(policyFunction(item, null, null, context, info))
+            }
+
+            for (const result of await Promise.all(promises)) {
```

Comment:
I would execute the policies serially.

From the standard:

> The order in which directives appear may be significant, including repeatable directives.

https://github.com/graphql/graphql-spec/blame/main/spec/Section%203%20--%20Type%20System.md#L1971-L1972

Reply:
done